File: spec/javascript/test_enum_negative.js
var assert = require('assert');
var testHelper = require('testHelper');

testHelper('EnumNegative', 'src/enum_negative.bin', function(r, EnumNegative) {
  assert.equal(r.f1, EnumNegative.Constants.NEGATIVE_ONE);
  assert.equal(r.f2, EnumNegative.Constants.POSITIVE_ONE);
});

File: spec/python/test_enum_negative.py
import unittest

from enum_negative import EnumNegative

class TestEnumNegative(unittest.TestCase):
    def test_enum_negative(self):
        r = EnumNegative.from_file("src/enum_negative.bin")

        self.assertEqual(r.f1, EnumNegative.Constants.negative_one)
        self.assertEqual(r.f2, EnumNegative.Constants.positive_one)
